src/App.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * App
+ * Define a Restful Webservice and routes controller requests
+ */
+
+import * as bodyParser from "body-parser";
+import * as express from "express";
+import * as core from "express-serve-static-core";
+import * as http from "http";
+import { CONFIG } from "./Configuration";
+import { NotificationController } from "./controllers/NotificationController";
+import { NotificationSubscriptionRequestType } from "./enums/NotificationSubscriptionType";
+import { logger } from "./utils/Logger";
+
+// Define server and routes
+export class App {
+  private readonly app: core.Express;
+  private readonly server: http.Server;
+
+  public constructor() {
+    this.app = express();
+    this.setGlobalSettings();
+    this.setServerRoutes();
+    this.server = http.createServer(this.app);
+  }
+
+  public startServer(): boolean {
+    logger.info("Starting Proxy PagoPa Server...");
+    this.server.listen(this.getServerPort());
+    this.server.on("error", this.onError);
+    return true;
```

Comment:
it looks like this function always returns true. 
do not return `public startServer(): void` is a better signature